api/proto/metrics/metrics.proto

```diff
@@ -144,6 +145,27 @@ message appMetric {
   repeated appDiskMetric disk = 6;
 }
 
+message logMetric {
+  uint64 numDeviceEventsSent = 1;
+  uint64 numDeviceBundlesSent = 2;
+  uint64 numAppEventsSent = 3;
+  uint64 numAppBundlesSent = 4;
+  uint64 num4xxResponses = 5;
```

Comment:
For app? For device? For both?
I need the algebra for these counters to understand what checks one can do using them.

Reply:
This counter includes both App & device 4xx response counts. Should I make them separate?

Reply:
It would be nice to have the algebra for the counters in comments in the proto files.
That makes it clear how things are counted when there are drops, 4xx, etc

Reply:
Will add in comments.

Reply:
done